src/game/models/player.rs

```diff
@@ -39,7 +39,10 @@ const THRUST_INCREMENT: f64 = 5.0;
 impl Player {
     pub fn new(window_size: Size) -> Player {
         Player {
-            pos: Vector { x: 10.0, y: 10.0 },
+            pos: Vector {
+                x: window_size.width as f64 / 2.0,
```

Comment:
:man_facepalming: I even saw that earlier and knew it was right. I must have thought I was in a different module when I wrote this.